README.md

```diff
@@ -84,3 +86,25 @@ helm upgrade ls-wg-system-metrics cribl/logstream-workergroup -n cribl-stream
 # Support
 
 Our community supports all items in the Cribl Helm repository – Please join our [Slack Community](https://cribl.io/community/)!
+
+# Running unit tests
```

Comment:
You read my mind!  This looks great and even works.  : )

![image](https://github.com/user-attachments/assets/bac25f68-eac0-4b4b-a75d-b0da0b12a7c3)